repl/appengine/README.md

```diff
@@ -0,0 +1,31 @@
+# CEL REPL Web
+Provides a simple web interface for working with the CEL REPL.
+
+The two main components are 1) a Go application that implements a service
+wrapping the REPL and serving static web content 2) an angular based web page
+that provides an interface around the API.
+
+The REPL service is stateless -- it initializes a new REPL instance and applies
+the requested commands in order on each request.
+
+## Development
+
+To run the application in development, run the angular CLI builder in watch mode
```

Comment:
We should specify that NodeJS is required. Angular CLI is optional for the purposes of running the app locally.